README.md

```diff
@@ -70,6 +70,7 @@ Full list of the changeable parameters
 
 | NAME                                | DESCRIPTION                                                                     | TYPE     | REQUIRED | DEFAULT               | OPTIONS                                  |
 | ----------------------------------- | ------------------------------------------------------------------------------- | -------- | -------- | --------------------- | ---------------------------------------- |
+| `github-api-url`                      | The Github API endpoint. Override for Github Enterprise usage.                           | `string` | `true`   | `https://api.github.com` |                                          |
```

Comment:
The CI is stacked with queue state, but this formatting should be blocked by dprint.
Could you format with `dprint fmt`?

Reply:
I think the prettier plugin for `dprint` doesn't quite agree with my system so it refuses to run. I reformatted manually for now while I figure that out.

Reply:
Ran prettier manually with `prettier -w --single-quote --print-width 120 README.md`

There was one unrelated change which I ignored.

Reply:
dprint is not a prettier plugin, it is a new replacement of prettier. Both have different rules and behaviors.

Providing nix and dockerfile in this repo https://github.com/kachick/wait-other-jobs/blob/f45d69523afdb4f239c354c4dddc4b2c185b8ea5/CONTRIBUTING.md#setup, however you can use it with another way https://dprint.dev/install/

Reply:
Right, I am on NixOS & have direnv flake integration. But:
```$ dprint fmt
Error resolving plugin https://plugins.dprint.dev/prettier-0.27.0.json: Error starting /home/<username>/.cache/dprint/cache/plugins/dprint-plugin-prettier/0.27.0/x86_64/dprint-plugin-prettier with args [--parent-pid 1458478 --init]. No such file or directory (os error 2)
```

This is because:
```
$ ldd /tmp/dprint-plugin-prettier
	linux-vdso.so.1 (0x00007ffe583d8000)
	libgcc_s.so.1 => /nix/store/rfckdjskd983ylf05jm9mlsw7y618hyr-xgcc-12.3.0-libgcc/lib/libgcc_s.so.1 (0x00007ff41e73e000)
	libpthread.so.0 => /nix/store/ld03l52xq2ssn4x0g5asypsxqls40497-glibc-2.37-8/lib/libpthread.so.0 (0x00007ff41e739000)
	libm.so.6 => /nix/store/ld03l52xq2ssn4x0g5asypsxqls40497-glibc-2.37-8/lib/libm.so.6 (0x00007ff41e659000)
	libdl.so.2 => /nix/store/ld03l52xq2ssn4x0g5asypsxqls40497-glibc-2.37-8/lib/libdl.so.2 (0x00007ff41e654000)
	libc.so.6 => /nix/store/ld03l52xq2ssn4x0g5asypsxqls40497-glibc-2.37-8/lib/libc.so.6 (0x00007ff41e46e000)
	/lib64/ld-linux-x86-64.so.2 => /nix/store/ld03l52xq2ssn4x0g5asypsxqls40497-glibc-2.37-8/lib64/ld-linux-x86-64.so.2 (0x00007ff420e6c000)
```

It's assuming `/lib64/ld-linux-x86-64.so.2` exists. Which means I need to use `patchelf` to run that plugin. But I'd need to figure out how to get that integrated into dprint to patch the plugin before it's run.

Reply:
Found a workaround so added a commit with (re)formatting